helm/examples/metrics/elasticsearch_metrics.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Defines a elastic sink for reporting metrics to an ElasticSearch instance
+# Settings:
+#   type                                - sink type (must be elastic for ElasticSearch support)
+#   name                                - name for the sink instance
+#   settings.elasticHost                - url for the ElasticSearch instance
+#   settings.indexName                  - ElasticSearch index name where metrics are indexed
+#   settings.countMetricSuffix          - suffix for count metrics
+#   settings.gaugeMetricSuffix          - suffix for gauge metrics
+#   settings.histogramMetricSuffix      - suffix for histogram metrics
+#
+# If not overridden, the following suffixes are used by default:
+#   countMetricSuffix: count
+#   gaugeMetricSuffix: gauge
+#   histogramMetricSuffix: histogram
+
+global:
+  metrics:
+    sinks:
+    - type: elastic
+      name: elasticsink
+      settings:
+        elasticHost: http://localhost:9200,
```

Comment:
perhaps too picky, but "Host" implies the dns entry only, no protocol, port, etc.